src/components/Log-In-form/index.jsx

```diff
@@ -32,10 +31,11 @@ return (
           <Link to = "/Signupform">
             <button className="Login__btn__Signin"><b>Sign up</b></button>
           </Link>
-      </form>
+        </form>
+        </div>
       </div>
-    </div>
-  </section>
-)
-}
-export default Loginform;
\ No newline at end of file
+    </section>
+  );
+};
```

Comment:
This semi-colon is not necesary.

Reply:
ESLint asked for the semi-colon in the individual assessment